cover.exs

```diff
@@ -0,0 +1,29 @@
+#!bin/elixir
+
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2021 The Elixir Team
+
+# TODO: Move this file in an acceptable location
+
+cover_dir = __ENV__.file |> Path.dirname() |> Path.join("cover")
+coverdata_inputs = cover_dir |> Path.join("{exunit,eunit}_*.coverdata") |> Path.wildcard()
```

Comment:
I am wondering if we should remove the eunit coverdata. eunit is mostly legacy tests and we are slowly converting them to exunit. This could actually help speed the process up, so I would discard it.

Reply:
This seems to work fine without a lot of extra effort. But If you prefer to not include it, I can also remove it.

Reply:
Can you please compare the reports with and without it and let us know if it differs by much?

Reply:
Yes, so let's nuke it. Although the number is too low and I don't think it is accurate. Well, some of our tests shell out and I assume those are not accounted... but it may be none of the compiled code is accounted either? Perhaps we may need to change it so the Makefile loads the coverage file before we load any of the test files.

Reply:
Ok, let me try to address all the failing tests first so that it runs through cleanly and then we can also see how to deal with all that.

Reply:
Nah, nevermind, we load it early enough according to this code: https://github.com/elixir-lang/elixir/blob/main/Makefile#L55